packages/vm/lib/evm/opcodes/EIP2200.ts

```diff
@@ -12,82 +12,66 @@ import { trap } from './util'
  * @param {Buffer}   value
  */
 export function updateSstoreGasEIP2200(runState: RunState, found: any, value: Buffer, key: Buffer) {
-  if (runState._common.gteHardfork('istanbul')) {
-    const original = found.original
-    const current = found.current
-    // Fail if not enough gas is left
-    if (
-      runState.eei.getGasLeft().lten(runState._common.param('gasPrices', 'sstoreSentryGasEIP2200'))
-    ) {
-      trap(ERROR.OUT_OF_GAS)
-    }
+  const original = found.original
+  const current = found.current
```

Comment:
personal style nit ^_^
```suggestion
  const { original, current } = found
```